packages/dapp-svelte-wallet/api/test/test-lib-wallet.js

```diff
@@ -786,11 +789,8 @@ test('lib-wallet addOffer for autoswap swap', async t => {
       simoleanBundle.mint.mintPayment(simoleanBundle.amountMath.make(1000)),
     );
 
-    const { publicAPI } = await E(zoe).getInstanceRecord(
-      autoswapInstanceHandle,
-    );
-
-    /** @type {Issuer} */
+    /** @type {{ getLiquidityIssuer(): Issuer, makeSwapInvite(): Invitation }} */
+    const publicAPI = await E(zoe).getPublicFacet(autoswapInstanceHandle);
```

Comment:
This is a small nit, but we are doing away with the name `InstanceHandle` in favor of `instance`

Reply:
Point taken.  The renaming will happen next time we break compatibility with the Dapps.